### PR TITLE
Fix a11y lint in BasicLtiLaunchApp

### PR DESCRIPTION
**Pull request by LMS007**
- Add title on iframe
- Add LMS Grader mode axe test (as this felt reasonable) Note: There are also a number of Dialogs that popup inside BasicLtiLaunchApp that are not reflected in `checkAccessibility` here, as Dialog shall have its own accessibility tests that reflect those relative states.

### CODE REVIEW

File: lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js [hunk 1]
```diff
@@ -239,11 +239,28 @@ describe('BasicLtiLaunchApp', () => {
...
-      content: () => renderLtiLaunchApp(),
-    })
+    checkAccessibility([
+      {
+        content: () => renderLtiLaunchApp(),
+      },
+      {
+        name: 'LMS grader mode',
+        content: () => {
+          // Override the fake config for this one test.
```
Comment (LMS007) [inline lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js]: Note: I checked checkAccessibility, looks like order honored, which if for any reason changes, then this test would not reflect its intent.

We may want to implement a beforeEach/afterEach sorta thing with an array of scenarios?

Comment (lyzadanger) [reply lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js]: > Note: I checked checkAccessibility, looks like order honored, which if for any reason changes, then the previous test may not reflect its intent.

Sorry, not following—could you rephrase?

Comment (lyzadanger) [inline lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js]: Can you expand this comment to explain _why_ the override is happening?

File: lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js [hunk 2]
```diff
@@ -189,13 +189,12 @@ export default function BasicLtiLaunchApp() {
...
   if (ltiLaunchState.state === 'fetched-url') {
     const iFrame = (
-      // FIXME-A11Y
-      // eslint-disable-next-line jsx-a11y/iframe-has-title
       <iframe
         width="100%"
         height="100%"
         className="js-via-iframe"
         src={ltiLaunchState.contentUrl}
+        title="Course content with Hypothesis annotation viewer"
```
Comment (lyzadanger) [inline lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js]: This looks good; and good job on following correct sentence-case convention!

File: lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js (hunk 1 above)
Comment (LMS007) [reply lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js]: @lyzadanger 
So, because these act like more than one test (array of tests) they can and often need to have their own config. But there is no `beforeEach` equivalent to reset all the params before each test. So when we override a fake object like `fakeConfig`, then that won't change back before the next test. In the is case, it happens to be the last test so its okay, but that is not obvious! I think we'll run into this a little bit with the axe tests and we may need some thinking about how to solve it elegantly.
